skimage/filters/window.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from ..transform import warp
+from .._shared.utils import safe_as_int
+from scipy.signal import get_window as get_window1d
+
+
+def get_window(window, size, ndim=2, **kwargs):
+    """Return an n-dimensional window of a given size and dimensionality.
+
+    Parameters
+    ----------
```

Comment:
Here I would copy as many information from the scipy docstring as needed (that was one of your questions). In particular put all the names of the available windows. Let's acknowledge that we're all lazy :-).

Reply:
I will include the available windows; I think that is the only information that is really missing at the moment. One concern: if a window function is added/removed in `scipy.signal`, it won't necessarily show up here. Unless I add a test to look for the availability of all docstring-listed windows (should I do that?).